api/src/pcapi/routes/backoffice_v3/templates/custom_reimbursement_rules/list.html

```diff
@@ -0,0 +1,82 @@
+{% import "components/forms.html" as forms with context %}
+{% import "components/links.html" as links %}
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5"
+       data-toggle="filters"
+       data-toggle-id="custom-reimbursement-rules">
+    <h2 class="fw-light">Tarifs dérogatoires</h2>
+    <div class="col-2">
+      <div class="py-2">
+        <button type="submit"
+                class="btn btn-primary filters-toggle-button"
+                disabled>
+          <span class="spinner-border spinner-border-sm"
+                role="status"
+                aria-hidden="true"></span>
+          <span class="visually-hidden">Chargement...</span>
+        </button>
+      </div>
+    </div>
+    <div class="filters-container">{{ forms.build_filters_form(form, dst) }}</div>
+    <div>
+      {% if rows %}
+        <div class="d-flex justify-content-between">
+          <p class="lead num-results">{{ rows|length }} résultat{{ "s" if rows|length > 1 else "" }}</p>
+        </div>
+        <table class="table mb-4">
+          <thead>
+            <tr>
+              <th scope="col"></th>
+              <th scope="col">ID règle</th>
+              <th scope="col">Structure</th>
+              <th scope="col">SIREN</th>
+              <th scope="col">Lieu</th>
+              <th scope="col">Offre</th>
+              <th scope="col">Taux de remboursement</th>
+              <th scope="col">Montant remboursé</th>
+              <th scope="col">Sous catégorie</th>
```

Comment:
```suggestion
              <th scope="col">Sous-catégories</th>
```
car tu affiches une liste